conn_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"database/sql"
 	"database/sql/driver"
+	"fmt"
+	uuid "github.com/satori/go.uuid"
```

Comment:
you sort your imports in conn_go18.go, but not here. Can you please make them same?